### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by IanMadd**
### :nut_and_bolt: Description: What code changed, and why?

This replaces links to pages in the old Hab docs site with links to the current Hab docs pages
This also replaces Edit on GitHub links with a partial that generates the GitHub page link.

### :chains: Related Resources

chef/chef-web-docs#2775

### :white_check_mark: Checklist

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [ ] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Review by kagarmoe — APPROVED** [review]
tenor-203472165

File: components/docs-chef-io/content/automate/applications_setup.md [hunk 1]
```diff
@@ -248,11 +248,10 @@ If you are using a certificate signed by a trusted certificate authority instead
 For more information, check out this further explanation on how to [configure Builder to authenticate via Chef Automate](https://github.com/habitat-sh/on-prem-builder).
 ## Related Resources
 
-For more information, see the [Chef Habitat documentation](https://www.habitat.sh/docs).
+For more information, see the [Chef Habitat documentation]({{< relref "/habitat" >}}).
 In particular, see the entries on:
 
-* [Chef Habitat Best Practices](https://www.habitat.sh/docs/best-practices/)
```
Comment (msorens) [inline components/docs-chef-io/content/automate/applications_setup.md:254]: This one got dropped; was that intentional?

**Review by msorens — COMMENTED** [review]
Would be useful (especially if you want anyone other than docs-team to review) to show a "before" and "after" screenshot of how the links are getting rendered.

File: components/docs-chef-io/content/automate/applications_setup.md (hunk 1 above)
Comment (IanMadd) [reply components/docs-chef-io/content/automate/applications_setup.md:254]: That page doesn't exist anymore and now redirects to docs.chef.io/habitat, which is the same as the link on line 251.

**Pull request merged**